### DISCUSSION TITLE
Results of the phase field tutorial

### DISCUSSION THREAD

**Discussion by wowodejiajia** [idaholab/moose#21441] [Q&A Modules: Phase field] [answered]
Dear MOOSE experts,

I follow the tutorial to get the results. There are many result files from the third step. Why are there so many out.e-s00x files?
 Which file is the final result?

Any suggestions or recommendations to fix the problem would be greatly appreciated.

Thank you
![s3](https://user-images.githubusercontent.com/85658962/176137834-2f9dbe66-b99c-4872-a290-3c6081c03ff2.png)

**Comment by mangerij** [comment, marked as answer]
These are from the adaptive mesh refinement (AMR). ParaView can open them all at once, just open the s3_decomp_out.e file and it will read all of those and you will see your mesh changing as a function of time in ParaView. If you turn off the AMR, you will only get one mesh file for a serial mesh.

**Reply by wowodejiajia** [reply]
I did what you said and the result was what you said. Thank you!
